modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -138,6 +139,7 @@ public class ApiClient {
      */
     public ApiClient setHttpClient(OkHttpClient httpClient) {
         this.httpClient = httpClient;
+        addProgressInterceptor();
```

Comment:
Note that if multiple `ApiClient`s are created with the same underlying `OkHttpClient` this will create duplicate progress updates for each request.  However, from reading the rest of the class (e.g. the `setDebugging()` method) I concluded that this is not a supported use case.